tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -189,6 +189,19 @@ Then('file/folder {string} should be listed in the trashbin on the webUI', funct
   return client.page.FilesPageElement.filesList().waitForFileVisible(file)
 })
 
+Then('the deleted elements should be listed in the trashbin on the webUI', function () {
```

Comment:
this function is not needed, can use `Then('file/folder {string} should not be listed on the webUI', function (folder) {`